backend/client.go

```diff
@@ -59,6 +59,7 @@ func (c *backendClient) ScheduleNewOrchestration(ctx context.Context, orchestrat
 
 	tc := helpers.TraceContextFromSpan(span)
 	e := helpers.NewExecutionStartedEvent(req.Name, req.InstanceId, req.Input, nil, tc)
+	e.EventType.(*protos.HistoryEvent_ExecutionStarted).ExecutionStarted.ScheduledStartTimestamp = req.ScheduledStartTimestamp
```

Comment:
Please add this as a parameter to the `helpers.NewExecutionStartedEvent` rather than doing this casting.